src/main/java/bob/Bob.java

```diff
@@ -0,0 +1,67 @@
+package bob;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import bob.command.Command;
+import bob.command.ExitCommand;
+import bob.exception.InvalidInputException;
+import bob.parser.Parser;
+import bob.storage.Storage;
+import bob.task.TaskList;
+import bob.ui.Ui;
+
+/**
+ * Bob is the main class for the application.
+ */
+public class Bob {
+    private TaskList tasks;
+    private Storage storage;
+    private Ui ui;
+
+    /**
+     * Constructs Bob.
+     *
+     * It constructs and initialises tasks, storage and ui.
+     *
+     * @param filePath the file path to load tasks from and save tasks to.
+     */
+    public Bob(String filePath) {
+        this.storage = new Storage(filePath);
+        this.ui = new Ui();
+        try {
+            tasks = this.storage.loadTaskList();
+        } catch (IOException e) {
+            ui.showError("Error loading tasks from file, using empty task list: " + e.getMessage());
+            tasks = new TaskList(this.storage);
+        } catch (InvalidInputException e) {
+            ui.showError("Error loading tasks from file, using empty task list: " + e.getMessage());
+            tasks = new TaskList(this.storage);
+        }
+
+    }
+
+    /**
+     * Main run loop for the application.
```

Comment:
```suggestion
     * Runs the main loop for the application.
```
Based on the coding standard, the first sentence of method header comments should start in the form `Returns ...`, `Sends ...`, `Adds ...`